circle.yml

```diff
@@ -7,9 +7,10 @@ dependencies:
 test:
   pre:
     - yarn license-check
+  override:
     - yarn eslint
     - yarn flow -- --quiet
-  override:
-    - yarn test:quiet -- --coverage --runInBand
+    - yarn test -- --coverage --runInBand
+    - yarn build:quiet
```

Comment:
I added the build step here because otherwise we don't have it anymore. And we need to know if the app still builds :)